_inc/client/plans/single-product-backup.jsx

```diff
@@ -0,0 +1,185 @@
+/**
+ * External dependencies
+ */
+import React, { useState } from 'react';
+import { translate as __ } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import { AccentedCard } from './accented-card';
+import Gridicon from 'components/gridicon';
+
+export function SingleProductBackup( props ) {
+	const { sitePlans } = props;
+
+	const [ selectedBackupType, setSelectedBackupType ] = useState( 'real-time' );
+
+	return (
+		<>
+			<h1 className="plans-section__header">Single Products</h1>
+			<h2 className="plans-section__subheader">Just looking for backups? We've got you covered.</h2>
+			<div style={ { display: 'flex', justifyContent: 'center', marginBottom: '10px' } }>
+				<AccentedCard>
+					{ {
+						header: <SingleProductBackupHeader sitePlans={ sitePlans } />,
+						body: (
+							<SingleProductBackupBody
+								selectedBackupType={ selectedBackupType }
+								setSelectedBackupType={ setSelectedBackupType }
+								sitePlans={ sitePlans }
+							/>
+						),
+					} }
+				</AccentedCard>
+			</div>
+		</>
+	);
+}
+
+function SingleProductBackupHeader( props ) {
+	const { sitePlans } = props;
+
+	return (
+		<div
+			style={ {
+				display: 'flex',
+				flexDirection: 'row',
+				justifyContent: 'space-between',
+				alignContent: 'center',
+			} }
+		>
+			<h3>{ __( 'Jetpack Backup' ) }</h3>
+			{ sitePlans && (
+				<PlanPriceDisplay
+					dailyPrice={ sitePlans[ 'daily-backup' ].price.yearly.text }
+					yearlyPrice={ sitePlans[ 'realtime-backup' ].price.yearly.amount }
+				/>
+			) }
+		</div>
+	);
+}
+
+export function PlanPriceDisplay( props ) {
+	const { dailyPrice, yearlyPrice } = props;
+	const perYearPriceRange = `${ dailyPrice }-${ yearlyPrice } /year`;
+
+	return (
+		<div
+			style={ {
+				display: 'flex',
+				flexDirection: 'row',
+				justifyContent: 'space-between',
+				alignContent: 'center',
+			} }
+		>
+			<SlashedPrice />
+			<div className="plans-price__container">
+				<span className="plans-price__span">{ perYearPriceRange }</span>
+			</div>
+		</div>
+	);
+}
+
+function SlashedPrice() {
+	return (
+		<div className="slashed-price__container" style={ { marginRight: '14px' } }>
+			<div className="slashed-price__slash"></div>
+			{ /* TODO: get this from an API or calculate, currently unsure how to get this increased price */ }
+			<div className="slashed-price__price">{ '$15-25' }</div>
+		</div>
+	);
+}
+
+function SingleProductBackupBody( props ) {
+	const { selectedBackupType, setSelectedBackupType, sitePlans } = props;
+
+	const upgradeLinks = {
```

Comment:
We typically use a redirect here, so as to select the right site as you land on WordPress.com.

As an example, see:
https://jetpack.com/redirect/?source=plans-premium&site=mysite.com&u=1
`getUpgradeUrl` can be helpful to build those.

Reply:
I had the upgrade links incorrect before, they are now switched to `https://wordpress.com/checkout/${ siteRawUrl }/professional` and `https://wordpress.com/checkout/${ siteRawUrl }/premium`. However I could also make it a redirect if that is better. What do you think?

Reply:
I believe the redirects are preferred as they add some tracking elements.